### PR TITLE
Add oAuth2 security, get services by animal method

### PR DESCRIPTION
**Pull request by shalauniou**
Should be merged after https://github.com/Andrei-Khadziukou/petclinic-plugins/pull/13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 * Add `management.security.enabled: false` property into `application.yml` file if you need to connect to secured endpoints from actuator (don't commit that configuration) 
 
 # Prepare plugins
-Download the petclinic-plugins project and run: gradle publishToMavenLocal (publishing plugins to local maven repository)
+Download the petclinic-plugins and read readme.md
```
Comment (Andrei-Khadziukou) [inline README.md]: What does that mean 'download'?

File: code-quality/checkstyle/checkstyle-suppressions.xml [hunk 2]
```diff
@@ -8,6 +8,8 @@
     <suppress files=".*Application.java" checks="HideUtilityClassConstructor"/>
 
     <suppress files="[\\/]model[\\/].*.java" checks="JavadocMethod"/>
+    <suppress files="[\\/]config[\\/].*.java" checks="JavadocMethod"/>
```
Comment (Andrei-Khadziukou) [inline code-quality/checkstyle/checkstyle-suppressions.xml]: Better to disable such checks in the code if it is possible (add TODO and issue in repo to solve it)

Comment (shalauniou) [reply code-quality/checkstyle/checkstyle-suppressions.xml]: Yes, it is an example how it works. I asked @andrei-astaschenko earlier to add such documentation, I think he does it better as he implemented this functionality. TODO will be added

File: README.md (hunk 1 above)
Comment (shalauniou) [reply README.md]: It might mean clone, pull, copy and so on

Comment (Andrei-Khadziukou) [reply README.md]: Good... Where can the person who is not familiar with our projects find the project with plugins?

**Pull request merged**